Doc/deprecations/pending-removal-in-3.16.rst

```diff
@@ -8,3 +8,6 @@ Pending Removal in Python 3.16
 * :mod:`symtable`:
   Deprecate :meth:`symtable.Class.get_methods` due to the lack of interest.
   (Contributed by Bénédikt Tran in :gh:`119698`.)
+
+* :class:`!shutil.ExecError` is deprecated:
+  use :class:`RuntimeError` instead.
```

Comment:
For consistency with the other entries:

```suggestion

* :mod:`shutil`: Deprecate :class:`!shutil.ExecError`, which hasn't
  been raised by any :mod:`!shutil` function since Python 3.4. It's
  now an alias for :exc:`RuntimeError`.
```